src/pc/gfx/gfx_citro3d.c

```diff
@@ -710,15 +744,14 @@ void gfx_citro3d_frame_draw_on(C3D_RenderTarget* target)
 static void gfx_citro3d_draw_triangles_helper(float buf_vbo[], size_t buf_vbo_len, size_t buf_vbo_num_tris)
 {
 #ifdef ENABLE_N3DS_3D_MODE
-    Mtx_Identity(&projection);
+    Mtx_Identity(&projection); // this doesn't seem needed
```

Comment:
I think this is to ensure if we drew something 3D then 2D we reset the projection matrix?